src/OctoshiftCLI/AdoClient.cs

```diff
@@ -43,14 +43,15 @@ public virtual async Task<string> GetAsync(string url)
             return content;
         }
 
-        public virtual async Task<string> PostAsync(string url, HttpContent body)
+        public virtual async Task<string> PostAsync(string url, string body)
```

Comment:
Not sure if it needs to be done in this PR but it would be better to follow our pattern and pass in an `object` body just like we did in `GitHubClient` to be able to pass in anonymous objects.

Reply:
I plan to do this, but in another PR